languages/javascript/exercises/concept/closures/.docs/instructions.md

```diff
@@ -0,0 +1,54 @@
+Your design company has primarily been working with CSS transformations to build webpages. After some discussion, a decision is made
+to start using JavaScript to perform some of the calculations dynamically. Some of your teammates are less experienced with JavaScript,
+so you decide to use a function closure to create reusable transformation for `{x, y}` coordinate pairs.
+
+## 1. Translate the coordinates
+
+Create a function which returns a function making use of a closure to perform a repeatable 2d translation of a coordinate pair.
```

Comment:
Could you add the name of the function to implement to the instructions?

```suggestion
Implement the `moveCoordinatesRight2Px` function which returns a function making use of a closure to perform a repeatable 2d translation of a coordinate pair.
```

p.s. this comment also applies to the other tasks
p.p.s. this comment also applies to several other exercises in the javascript track (the `basics` exercise is correct)